app/api/endpoints/chat.py

```diff
@@ -21,8 +21,11 @@ async def send_message(
     template_name: str = Query(
         "chat_default",
         description="The template name without suffix, e.g. chat_default, default to 'chat_default' for gchat"),
-    environment: Dict[str, Any] = Body(
-        {"body": ""},
+    environment: Union[schemas.DictBody, schemas.TxtBody] = Body(
+        ...,
+        example={
+            "body": "SAMPLE MESSAGE USE THE DEFAULT CHAT TEMPLATE, REPLACE IT OR USE DIFFERENT TEMPLATE AND CONTENT."
+        },
         description="The body values for parse with the template"),
```

Comment:
I would mention Jinja here and 'chat_default' with a link to https://github.com/waynesun09/notify-service/blob/main/app/templates/build/chat_default.jinja